x-pack/legacy/plugins/integrations_manager/server/integrations/data.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  SavedObjectsBulkGetObject,
+  SavedObjectsClientContract,
+} from 'src/core/server/saved_objects';
+import { InstallationSavedObject, Installable } from '../../common/types';
+import { SAVED_OBJECT_TYPE } from '../../common/constants';
+import * as Registry from '../registry';
+
+export async function getIntegrations(client: SavedObjectsClientContract) {
```

Comment:
These functions all accept a saved object client. Mainly because we get the client from the request, but it also means we can inject a client. Figure we’ll come back to this over time and [when the client lands in core](https://github.com/elastic/kibana/pull/39891) .

Reply:
So happy to have TS here...